build/android-build.py

```diff
@@ -1,120 +1,100 @@
-#!/usr/bin/python
+#!/usr/bin/env python
```

Comment:
Using shebang like this ensures that the `python` that is actually on PATH is used.

As an example, it is a common way for macOS users to install python using homebrew, and homebrew doesn't put files to system directories.